src/test/java/edu/cornell/kfs/module/purap/businessobject/xml/fixture/IWantTransactionLineFixture.java

```diff
@@ -0,0 +1,51 @@
+package edu.cornell.kfs.module.purap.businessobject.xml.fixture;
+
+import java.math.BigDecimal;
+
+import edu.cornell.kfs.module.purap.businessobject.xml.IWantTransactionLineXml;
+import edu.cornell.kfs.module.purap.businessobject.xml.IWantXmlConstants.IWantAmountOrPercentTypeXml;
+
+public enum IWantTransactionLineFixture {
+
+    TRANSACTION_LINE_TEST("chart", "account", "object code", "sub account", "sub object code", "project code",
+            "reference id", BigDecimal.valueOf(666.66), IWantAmountOrPercentTypeXml.A);
```

Comment:
Another spot where the `BigDecimal` wrapping could be moved to the constructor body.